themes/hugo-bootstrap-5/layouts/tcf/tcf-main.html

```diff
@@ -0,0 +1,356 @@
+<!DOCTYPE html>
+<html lang="{{ .Lang }}">
+  <head>
+    {{ partial "head-meta" . }}
+
+
+    <link rel="canonical" href="{{ .Permalink }}" />
+
+    <title>UBC Technical Career Fair 2024</title>
+
+    <!-- Bootstrap CSS -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/5.2.2/css/bootstrap.min.css"
+      integrity="sha512-CpIKUSyh9QX2+zSdfGP+eWLx23C8Dj9/XmHjZY2uDtfkdLGo0uY12jgcnkX9vXOgYajEKb/jiw67EYm+kBf+6g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer" />
+    <script
+      src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js"
+      integrity="sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3"
+      crossorigin="anonymous"></script>
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.min.js"
+      integrity="sha384-cuYeSxntonz0PPNlHhBs68uyIAVpIIOZZ5JqeqvYYIcEL727kskC66kF92t6Xl2V"
+      crossorigin="anonymous"></script>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.2/font/bootstrap-icons.css" />
+
+    <link href="{{ "css/style.css" | relURL }}" rel="stylesheet" />
+    <link href="{{ "css/tcf.css" | relURL }}" rel="stylesheet" />
+
+    {{ range .AlternativeOutputFormats -}}
+      <link
+        rel="{{ .Rel }}"
+        type="{{ .MediaType.Type }}"
+        href="{{ .Permalink | safeURL }}" />
+    {{ end -}}
+    {{ template "_internal/google_analytics_async.html" . }}
+    {{ partial "head-custom.html" . }}
+  </head>
+  <body>
+    <div class="dropdown nav-sticky">
+      <button
+        class="btn btn-light dropdown-toggle position-absolute m-3 small-shadow"
+        type="button"
+        data-bs-toggle="dropdown"
+        aria-expanded="false">
+        <i class="bi-list"></i>
+      </button>
+      <ul class="dropdown-menu">
+        <li><a class="dropdown-item" href="/tcf#about">About</a></li>
+        <li><a class="dropdown-item" href="/tcf/companies">Companies</a></li>
+        <li><a class="dropdown-item" href="/tcf#events">Q&A Events</a></li>
+        <li><a class="dropdown-item" href="/tcf#event-info">Event Info</a></li>
+        <li><a class="dropdown-item" href="/tcf#faq">FAQ</a></li>
+      </ul>
+    </div>
+    <div class="hero-header coming-soon">
+      <div class="hero-header-content py-5">
+        <div class="mx-auto py-5 my-5">
+          <p class="hero-header-text">Coming to UBC January 18, 2024</p>
+          <h1 class="display-1 fw-bold">Technical Career Fair 2024</h1>
+          <p class="hero-header-text">More details coming soon...</p>
+          <p>
+            To learn more about our event last year, see
+            <a href="/tcf2023/" style="color:white;"> TCF 2023 </a>
+          </p>
+          <p>Questions? Email vpx@ubccsss.org</p>
+        </div>
+      </div>
+    </div>
+    <!-- <div class="container" id="about">
```

Comment:
Same as above, let's delete this and re-reference old PRs when needed. (This gets processed out but just to keep the files clean.)